common/helpers/field.js

```diff
@@ -19,6 +20,25 @@ function mapReferenceDataToOption({ id, title, key, conditional }) {
   return option
 }
 
+function mapAssessmentQuestionToTranslation(item) {
+  const { category, key } = item
+  const translations = ['hint', 'label']
+  const translationStrings = {}
+
+  translations.forEach(translation => {
```

Comment:
Wonder if this needs a `forEach`.

Maybe we handle the label and hint individually where in the case of the _label_ not being present we use `title` as the fallback.

Reply:
What are your thoughts on something like this?

```js
function mapAssessmentQuestionToTranslation(item) {
  const { category, key, title } = item
  const fieldPath = `fields::${category}.items.${key}`
  const titleKey = `${fieldPath}.title`
  const hintKey = `${fieldPath}.hint`

  return {
    ...item,
    title: i18n.exists(titleKey) ? titleKey : title,
    hint: i18n.exists(hintKey) ? hintKey : undefined,
  }
}
```

Reply:
> What are your thoughts on something like this?

Sounds good to me. If it's urgent please feel free to take this over. If not I can pick it up when back.